matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/internal/LocalMatchFinder.java

```diff
@@ -53,6 +77,38 @@
     @Inject
     private MatchStorageManager matchStorageManager;
 
+    @Inject
+    private Provider<XWikiContext> provider;
+
+    private DateTimeFormatter dateFormatter = ISODateTimeFormat.dateTime().withZone(DateTimeZone.UTC);
+
+    private Date previousStartedTime;
+
+    private XWikiDocument prefsDoc;
+
+    @Override
+    public void initialize() throws InitializationException
+    {
+        XWikiContext context = this.provider.get();
+        this.previousStartedTime = null;
```

Comment:
i think style checker will complain about initializing to `null` since that is already the default value